README.md

```diff
@@ -1,2 +1,64 @@
-# configs
-Config files used to define parameters specific to compute environments at different Institutions.
+<img src="docs/images/nf-core-logo.png" width="400">
+
+# [nf-core/configs](https://github.com/nf-core/configs)
+[![Build Status](https://travis-ci.org/nf-core/configs.svg?branch=master)](https://travis-ci.org/nf-core/configs)
```

Comment:
There is no testing or code coverage for this repo, so no need for these badges..